app/views/claims/support/schools/users/check.html.erb

```diff
@@ -25,7 +29,13 @@
             </dd>
             <dd class="govuk-summary-list__actions">
               <ul class="govuk-summary-list__actions-list">
-                <li class="govuk-summary-list__actions-list-item"><a class="govuk-link" href="#">Change<span class="govuk-visually-hidden"></span></a></li>
+                <li class="govuk-summary-list__actions-list-item">
+                  <%= govuk_link_to(t(".change"),
+                    new_claims_support_school_user_path(
+                      params.permit(claims_user: {}).merge(school_id: @school.id),
```

Comment:
Same here

Reply:
resolved in [e56a8f2](https://github.com/DFE-Digital/itt-mentor-services/pull/136/commits/e56a8f27dbadac36b1c7b5800ec59c670bba23cb)